hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ReplicationManager.java

```diff
@@ -766,6 +773,32 @@ private void handleUnstableContainer(final ContainerInfo container,
 
   }
 
+  /**
+   * Check and update Container key count and used bytes based on it's replica's
+   * data.
+   */
+  private void checkAndUpdateContainerState(final ContainerInfo container,
```

Comment:
I think something along the lines of `updateContainerStatsFromReplicas` would be more descriptive, especially since container state (closed, etc.) is not being changed.